modules/install/pages/thp-disable.adoc

```diff
@@ -3,18 +3,24 @@
 :tabs:
 
 [abstract]
+--
 {description}
-THP must be disabled in order for Couchbase Server to function correctly on Linux.
+Since we tend to have more random, sparse data access, we allocate pages that can remain mostly empty.
+This leads to memory fragmentation as portions of memory are not used but still accounted for in the RSS.
+
+As a result, the data stored which we keep track of may be smaller while RSS can be significantly higher,
+leading to possible OOM kill.
+--
 
 In Linux operating systems, _huge pages_ is a feature that provides a way for the CPU and OS to create pre-allocated contiguous memory space, and which is designed to improve application performance.
-_Transparent huge pages (THP)_ is a Linux OS feature that automates the creation of contiguous memory space, and conceals much of the complexity of using actual huge pages on systems with large amounts of memory.
+_Transparent huge pages (THP)_ is a Linux OS feature that automates the creation of contiguous memory space and conceals much of the complexity of using actual huge pages on systems with large amounts of memory.
 
 THP is enabled by default in most Linux operating systems, and functions very well for most applications and processes.
 However, THP is detrimental to Couchbase's performance (as it is for nearly all databases that tend to have sparse rather than contiguous memory access patterns).
```

Comment:
We can add your new points from above here instead to explain the reasoning in detail.